ob-dart.el

```diff
@@ -40,38 +36,34 @@
 ;;     - ob-groovy.el
 ;;     - ob-R.el
 ;;     - ob-python.el
-
+;;
 ;;; Code:
 
 (require 'ob)
-(eval-when-compile (require 'cl))
```

Comment:
Although the modern form of the Common Lisp extension is imported as `cl-lib`, no symbols from it were actually in use here, so the import could be removed entirely.